docs/docs/index.md

```diff
@@ -1,96 +1,148 @@
 # Overview
 
-Dataview is a live index and query engine over your knowledge base. You can associate *data* (like tags, dates,
-snippets, numbers, and so on) with your markdown pages, and then *query* (like filter, sort, transform) this data. This
-is a simple but powerful idea:
+Dataview is a live index and query engine over your personal knowledge base. You can [**add metadata**](annotation/add-metadata.md) to your notes and **query** them with the [**Dataview Query Language**](queries/structure.md) to list, filter, sort or group your data. Dataview keeps your queries always up to date and makes data aggregation a breeze.
 
-- Track sleep schedules and habits by recording them in daily notes, and automatically create weekly tables of your
-  sleep schedule.
+You could
+
+- Track your sleep by recording it in daily notes, and automatically create weekly tables of your sleep schedule.
 - Automatically collect links to books in your notes, and render them all sorted by rating.
-- Automatically collect pages annotated with a given date, showing them in your daily note or elsewhere.
+- Automatically collect pages associated with today's date and show them in your daily note.
 - Find pages with no tags for follow-up, or show pretty views of specifically-tagged pages.
-- Create dynamic views which show upcoming birthdays or events, annotated with notes.
+- Create dynamic views which show upcoming birthdays or events recorded in your notes
+
+and many, many things more.
+
+!!! hint "Dataview gives you a fast way to search, display and operate on indexed data in your vault!"
+
+Dataview is highly generic and high performance, scaling up to hundreds of thousands of annotated notes without issue. 
+
+If the built in [query language](query/queries/) is insufficient for your purpose, you can run arbitrary
+JavaScript against the [dataview API](api/intro/) and build whatever utility you might need yourself, right in your notes.
+
+!!! info "Dataview is about displaying, not editing"
+    Dataview is meant for displaying and calculating data. It is no utility to edit data in your source files and will keep your notes always untouched (... except if you're checking a [Task](queries/query-types.md#task-queries) through Dataview.)
+
+## How to Use Dataview
+
+Dataview consists out of two big building blocks: **Data Indexing** and **Data Querying**. 
+
+!!! info "More details on the linked documentation pages"
+    The following sections should give you a broad overview about what you can do with dataview and how. Be sure to visit the linked pages to find out more about the individual parts.
```

Comment:
"broad" and "overview" seems redundant.

Reply:
You feel so? I think you can give broad or detailed overviews. I'll change it to "general overview", to point out we look at all aspects without going into detail.

Reply:
Feel free to reject the idea. I just have a habit of whittling down word counts lol.